src/platform/forms/tests/sub-task/basic-2-page.unit.spec.jsx

```diff
@@ -65,7 +65,7 @@ describe('the Supplemental Claims Sub-task', () => {
 
     fireEvent.click($('va-button[continue]', container), mouseClick);
     expect($('form[data-page="other"]', container)).to.exist;
-    expect($('a[download]', container)).to.exist;
+    expect($('va-link', container)).to.exist;
```

Comment:
It might be better to be more specific here, like `va-link[download]`

Reply:
@Mottie good call. Thank you!